### ISSUE TITLE
Feature request: Better `obs_impact_tool` documentation

### ISSUE THREAD

**Issue by josephCMY** [NCAR/DART#571]
### Use case  
Using `obs_impact_tool` with WRF-DART to turn off model increments for certain variables

### Is your feature request related to a problem?  
The documentation for `obs_impact_tool` only cover how to use the `obs_impact_tool` program. It is missing a description of how to use the output with `filter`. 

### Describe your preferred solution  
Add a blurb about what namelist options to specify in `input.nml` when calling `filter`.

**Comment by hkershaw-brown** [comment]
assim_tools_nml namelist options required o use the obs impact code during filter"

```
  adjust_obs_impact               = .true.
  obs_impact_filename             = 'control_impact_runtime.txt'

```

obs_impact_tool is used to create the  'control_impact_runtime.txt' file:
https://docs.dart.ucar.edu/en/latest/assimilation_code/programs/obs_impact_tool/obs_impact_tool.html

**Comment by hkershaw-brown** [comment]
add to quickbuild.sh

**Comment by hkershaw-brown** [comment]
fixed in #655
